.github/workflows/test.yml

```diff
@@ -31,3 +31,64 @@ jobs:
       - name: Test ddg2dnr
         working-directory: ./extension/packages/ddg2dnr
         run: npm test -- --grep Reference
+  BrowserServicesKit:
```

Comment:
I don't know github action syntax well enough to review, but @sammacbeth  did take a look and gave it a 👍 :) This is mostly copied from BSK repo, right?

Reply:
For the most part yes 👍